cmd/helmExecute.go

```diff
@@ -187,3 +189,9 @@ func parseAndRenderCPETemplate(config helmExecuteOptions, rootPath string, utils
 
 	return nil
 }
+
+type fileHandler interface {
+	FileExists(string) (bool, error)
+	FileRead(string) ([]byte, error)
+	FileWrite(string, []byte, fs.FileMode) error
```

Comment:
I assume this interface was introduced for better testability?
But why the change from [`os.FileMode`](https://github.com/SAP/jenkins-library/blob/master/pkg/piperutils/fileUtils.go#L29) to `fs.FileMode`?

Reply:
@LindaSieb What about this?

Reply:
I'm not sure why it was changed, I'm fine to change it back!

Reply:
https://github.com/SAP/jenkins-library/pull/4511/commits/8d7d30fdbd35836d79e1078df03219dc3bbe6ba0